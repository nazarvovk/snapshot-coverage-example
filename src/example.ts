export const example = () => {
  const test1 = 'this is a test function'

  const test2 = '<- when creating a snapshot with --coverage flag, this variable will not be called test2'
}
