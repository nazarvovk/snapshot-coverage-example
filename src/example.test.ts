import { example } from './example'

describe('example', () => {
  it('should match function snapshot', () => {
    expect(example.toString()).toMatchSnapshot()
  })
})
