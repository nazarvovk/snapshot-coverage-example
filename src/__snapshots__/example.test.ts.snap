// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example should match function snapshot 1`] = `
"()=>{
    const test1 = 'this is a test function';
    const test2 = '<- when creating a snapshot with --coverage flag, this variable will not be called test2';
}"
`;
